src/map/status.cpp

```diff
@@ -15256,11 +15260,15 @@ static int status_natural_heal(struct block_list* bl, va_list args)
 	if (flag&RGN_HP) {
 		// Interval to next recovery tick
 		rate = (int)(battle_config.natural_healhp_interval / (regen->rate.hp/100. * multi));
-		if (ud && ud->walktimer != INVALID_TIMER)
+		// Half recovery while moving only applies to players with certain traits
+		if (sd && ud && ud->walktimer != INVALID_TIMER)
 			rate *= 2;
-		// Homun HP regen fix (they should regen as if they were sitting (twice as fast)
+		// Homun HP regen fix (2 seconds instead of 6 seconds)
```

Comment:
For me having the absolute numbers here, when we talk about a "rate" is a little bit confusing.
Applies to all comments below.

Reply:
The absolute numbers are the official values. 
We just make it dynamic because we have config settings to adjust the official values.
I think this makes it clear what this adjustment was originally based on.